### DISCUSSION TITLE
How to put data to IDS from Lambda AWS in python?

### DISCUSSION THREAD

**Discussion by YogarajRethinasamy** [International-Data-Spaces-Association/DataspaceConnector#83] [Q&A] [answered]
How to put data to IDS from Lambda AWS in python

**Comment by juliapampus** [comment, marked as answer]
How to provide data as an offer to an IDS ecosystem using the Dataspace Connector is explained [here](https://international-data-spaces-association.github.io/DataspaceConnector/CommunicationGuide/v6/Provider).
